src/main/java/org/spongepowered/mod/SpongeGame.java

```diff
@@ -97,7 +101,24 @@ public String getImplementationVersion() {
 
     @Override
     public MinecraftVersion getMinecraftVersion() {
-        throw new UnsupportedOperationException(); // TODO
+	    MinecraftVersion mcvToReturn;
+
+	    // Get the MC version and protocol version from the MC server, then parse it so we can use it earlier
+	    ServerStatusResponse.MinecraftProtocolVersionIdentifier protocolVersionInfo = MinecraftServer.getServer().getServerStatusResponse().getProtocolVersionInfo();
```

Comment:
These lines can be removed. Just add a constructor in SpongeMinecraftVersion that accepts MinecraftProtocolVersionIdentifier so you can do the following

``` java
serverVersion = new SpongeMinecraftVersion(MinecraftServer.getServer().getServerStatusResponse().getProtocolVersionInfo());
``` java